bake.hcl

```diff
@@ -0,0 +1,116 @@
+variable "IMAGE_TAG" {
+  default = "4.x"
+}
+
+variable "DOCKERHUB_NAMESPACE" {
+  default = "singledigital"
+}
+
+variable "CONTEXT" {
+  default = "bay/images"
+}
+
+variable "LAGOON_IMAGE_VERSION" {
+  # Used to control which version of upstream Lagoon
+  # images the bay images are built from.
+  default = "latest"
```

Comment:
We could look to move this to a pinned version number to make sure that we have CAB control around the lagoon image inheritance process.